portfolio.css

```diff
@@ -0,0 +1,925 @@
+@font-face {
+  font-family: "Roboto";
+  src: url(fonts/Roboto-Regular.ttf);
+}
+
+@font-face {
+  font-family: "Crete Round";
+  src: url(fonts/CreteRound-Regular.ttf);
+}
+
+@font-face {
+  font-family: "Poppins";
+  src: url(fonts/Poppins-Regular.ttf);
+}
+
+@font-face {
+  font-family: "Inter";
+  src: url(fonts/Inter-VariableFont_slnt\,wght.ttf);
+}
+
+@font-face {
+  font-family: "InaiMathi";
+  src: url(fonts/Poppins-Bold.ttf);
+}
+
+html {
+  width: 100%;
+  height: 100%;
+  margin: 0;
+  padding: 0;
+  overflow-x: hidden;
+}
+
+body {
```

Comment:
- [ ] As the previous reviewer mentioned your design is not similar to the given one, please make the changes so the design is as close to the Figma design as possible

| Your design | Figma design |
|-----------------|---------------------|
| ![Screenshot_20220517_214118](https://user-images.githubusercontent.com/25227022/168906252-80c7c7ca-4f8f-4627-9fa0-2ab50769b34e.png)|![Screenshot_20220510_190024](https://user-images.githubusercontent.com/25227022/168906339-9a9dd023-42e5-41e7-86b8-e342ad52d858.png)|